Sources/Benchmark/Constants.swift

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Google LLC
```

Comment:
Consider renaming this file `StringConstants.swift` (if it's for large string constants)?

Reply:
I dropped the constants file for now, due to it having just a single constant. We can bring it back once it's more clear what to put there.

Reply:
SGTM.